### PR TITLE
chore: reduce dependencies

### PR DESCRIPTION
**Pull request by tsctx**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,8 +118,6 @@
     "jsfuzz": "^1.0.15",
     "mitata": "^0.1.6",
     "mocha": "^10.0.0",
-    "mockttp": "^3.9.2",
```
Comment (tsctx) [inline package.json:122]: This is not actually used.
https://github.com/search?q=repo%3Anodejs%2Fundici+mockttp&type=code

**Review by ronag — CHANGES_REQUESTED** [review]
mockttp ok, delay I don't see any point in re-implementing ourselves.

**Comment by tsctx** [comment]
> mockttp ok, delay I don't see any point in re-implementing ourselves.

I did so simply to replace it.

@ronag I made some changes, can you take a look?

**Comment by ronag** [comment]
Don't add more code :/. For dev deps it's fine to have dependencies.

File: package.json (hunk 1 above)
Comment (metcoder95) [reply package.json:122]: Thanks, I left it when working on the `RedirectHandler` and forget to remove it

**Comment by tsctx** [comment]
@ronag Is that all right with you?

**Review by metcoder95 — APPROVED** [review]

**Review by KhafraDev — APPROVED** [review]

**Pull request merged**